### PR TITLE
Change variant tooltip in pre-game

### PR DESCRIPTION
**Pull request by vEnhance**
Implement #1710 because I want this feature too :stuck_out_tongue:

### CODE REVIEW

File: client/src/lobby/nav.ts [hunk 1]
```diff
@@ -94,10 +146,11 @@ export const init = () => {
 
 const initTooltips = () => {
   const tooltips = [
-    'create-game',
-    'replay',
-    'resources',
-    'settings',
```
Comment (Zamiell) [inline client/src/lobby/nav.ts:100]: why are you renaming these

File: client/src/lobby/nav.ts [hunk 2]
```diff
@@ -57,6 +58,57 @@ export const init = () => {
...
+      if (event.key === 'Enter') {
+        event.preventDefault();
+        $('#change-variant-update').click();
+      }
+    });
+
+    // Update button trigger
+    $('#change-variant-update').on('click', () => {
+      const variantName = ($('#change-variant-dropdown').val() as string).trim();
+      console.log(variantName);
```
Comment (Zamiell) [inline client/src/lobby/nav.ts:97]: why is this here

Comment (vEnhance) [reply client/src/lobby/nav.ts]: mb

File: client/src/lobby/nav.ts (hunk 1 above)
Comment (vEnhance) [reply client/src/lobby/nav.ts:100]: I wanted to `initToolTips` also hit `nav-buttons-pregame-change-variant`

File: client/src/lobby/pregame.ts [hunk 3]
```diff
@@ -423,7 +423,7 @@ const drawPlayerBox = (i: number) => {
 };
 
 export const enableStartGameButton = () => {
-  // Enable or disable the "Start Game" button,
+  // Enable or disable the "Start Game" and "Change Variant" button,
   // depending on if we are the game owner and enough players have joined
```
[the remark this answers is not in the document]
Comment (vEnhance) [reply client/src/lobby/pregame.ts:427]: The dropdown is okay with just one player (leader) in pregame, and the code is written that way.

I can see the comment might be a little misleading as written --- feel free to reword as you see fit.

**Comment by vEnhance** [comment]
Anything else I should do for this PR?

File: client/src/lobby/nav.ts (hunk 1 above)
Comment (Zamiell) [reply client/src/lobby/nav.ts:100]: using a `games-` prefix implies that they are part of the game screen, when they are not
why not use a `lobby-` prefix

Comment (vEnhance) [reply client/src/lobby/nav.ts:100]: The `games-` prefix isn't by me, was there before, but I can rename them all to `lobby-`. Agree it makes more sense.

File: client/src/lobby/pregame.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (vEnhance) [reply client/src/lobby/pregame.ts]: 41ded0a clarifies the comment

File: client/src/lobby/nav.ts (hunk 1 above)
Comment (vEnhance) [reply client/src/lobby/nav.ts:100]: 18ce2e41dcfebb2467ee20a4cf317c861452502a does this renaming. this is a fairly deep change... hope it doesn't break anything (though it seems to work as far as my testing can see)

**Comment by Zamiell** [comment]
> Anything else I should do for this PR?

1) make the button the default css, e.g. black
2) center the button in the tooltip
